scripts/run_integration_tests_aarch64.sh

```diff
@@ -308,6 +308,17 @@ echo "Integration test on FDT finished with result $RES."
 if [ $RES -eq 0 ]; then
     # Test with EDK2 + ACPI
     cargo build --all --release $features_build_acpi --target $BUILD_TARGET
+    strip target/$BUILD_TARGET/release/cloud-hypervisor
+    strip target/$BUILD_TARGET/release/vhost_user_net
+    strip target/$BUILD_TARGET/release/ch-remote
+
+    # Enable KSM with some reasonable parameters so that it won't take too long
+    # for the memory to be merged between two processes.
+    sudo bash -c "echo 1000000 > /sys/kernel/mm/ksm/pages_to_scan"
+    sudo bash -c "echo 10 > /sys/kernel/mm/ksm/sleep_millisecs"
+    sudo bash -c "echo 1 > /sys/kernel/mm/ksm/run"
```

Comment:
Why did you add these? Are you using KSM?

Reply:
I simply copied these from lines above. I cannot remember why they were there. Should I remove them?

Reply:
Ah, got your meaning. You mean they are not needed for that ACPI test case, right?

Reply:
Yes that's what I meant :)